op-devstack/dsl/proofs/fault_dispute_game.go

```diff
@@ -31,8 +34,19 @@ func NewFaultDisputeGame(t devtest.T, require *require.Assertions, game *binding
 	}
 }
 
-func (g *FaultDisputeGame) MaxDepth() uint64 {
-	return contract.Read(g.game.MaxGameDepth()).Uint64()
+func (g *FaultDisputeGame) Addr() *common.Address {
+	// Pull To address info from a TypedCall
+	call := g.game.Status()
+	test := call.Test()
+	addr, err := call.To()
+	test.Require().NoError(err, "Failed to retrieve address from game")
```

Comment:
This is an awkward way to get the game address, but it doesn't seem to be accessible from `g.game` directly.